src/commands/deploy/showCoreToolsWarning.ts

```diff
@@ -5,16 +5,16 @@
 
 import { DialogResponses, IActionContext, nonNullValue, openUrl } from "@microsoft/vscode-azext-utils";
 import { MessageItem } from "vscode";
-import { getLocalFuncCoreToolsVersion } from "../../funcCoreTools/getLocalFuncCoreToolsVersion";
 import { FuncVersion, funcVersionLink, tryParseFuncVersion } from "../../FuncVersion";
+import { tryGetLocalFuncVersion } from "../../funcCoreTools/tryGetLocalFuncVersion";
 import { localize } from "../../localize";
 import { getWorkspaceSetting, updateGlobalSetting } from "../../vsCodeConfig/settings";
 
 export async function showCoreToolsWarning(context: IActionContext, runtimeVersion: FuncVersion, siteName: string): Promise<void> {
```

Comment:
I would advocate that a warning like this shouldn't block deploy regardless of error/success, meaning this call probably shouldn't be awaited:

https://github.com/microsoft/vscode-azurefunctions/blob/main/src/commands/deploy/deploy.ts#L75

Reply:
This seems fair enough. We show a warning, but we don't give any actional buttons that would cancel the deployment, so there doesn't seem to be a reason to stop the deployment. 

It's not a modal either, so it may not be obvious that's why your deployment is being blocked.